mne_bids/meg_bids.py

```diff
@@ -233,8 +235,8 @@ def raw_to_bids(subject_id, session_id, run, task, raw_fname, output_path,
         The path of the BIDS compatible folder
     events_fname : str
         The path to the events file.
-    event_id : dict
-        The event id dict
+    event_id : dict | str
+        The event id dict or json file path
```

Comment:
file path? I thought you give the dict as string. Otherwise, you need to read the file first and then use `json.loads`

Reply:
@choldgraf mentioned adding file path here https://github.com/mne-tools/mne-bids/issues/9#issuecomment-371050391. I added the file read [here](https://github.com/mne-tools/mne-bids/pull/31/files#diff-f5af84367dbb84da9caa996e06836846R93) but I can also add reading a string dict

Reply:
umm, I don't know. Depends on how big we expect `event_id` to be I guess